lib-database-entities/src/commonMain/kotlin/com/ustadmobile/lib/db/entities/StatementEntityWithDisplay.kt

```diff
@@ -4,7 +4,7 @@ import androidx.room.Embedded
 import kotlinx.serialization.Serializable
 
 @Serializable
-class StatementListReport : StatementEntity() {
+class StatementEntityWithDisplay : StatementEntity() {
```

Comment:
Should be StatementEntityWithDisplayDetails  - with display on it's own implies it has a monitor or something